### DISCUSSION TITLE
File History in Source Control

### DISCUSSION THREAD

**Discussion by DKhalil** [gitkraken/vscode-gitlens#2753] [Ideas]
Hello!

While I really enjoy the depth of the file history possible now in the Inspect tab, I mostly use the source control tab for navigating the repo and I really do miss having a general overview of the file history there. Is it possible to re-enable it there? It would be greatly appreciated.

**Comment by jkelroy** [comment]
Hello @DKhalil, it is possible to move views to another sidebar by dragging and dropping it.

https://github.com/gitkraken/vscode-gitlens/assets/27739784/372aac35-4a0e-4e6c-a44e-6c2e1dbdce7c

**Reply by sKopheK** [reply]
took me 10 minutes to remember that, it's not very straightforward how VSC visualizes that - basically forcing you to drop it in a new pane instead of an existing one
